package.json

```diff
@@ -21,10 +21,10 @@
     "fresh": "npm run clean && npm install && npm start",
     "lint": "jshint . --verbose --exclude-path=.jshintignore",
     "doc": "jsdoc -c .jsdocrc",
-    "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
-    "sia-import": "test -d app/dependencies/Sia || mkdir -p app/dependencies/Sia && cp $GOPATH/bin/siad app/dependencies/Sia",
-    "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
-    "sia-reimport": "(cd $GOPATH/src/github.com/NebulousLabs/Sia && make) && npm run sia-kill && npm run sia-import",
+    "s-repo": "go get -u github.com/NebulousLabs/Sia/...",
+    "s-import": "test -d app/dependencies/Sia || mkdir -p app/dependencies/Sia && cp $GOPATH/bin/siad app/dependencies/Sia",
+    "s-kill": "killall siad; while pgrep siad; do sleep .5; done;",
+    "s-reimport": "(cd $GOPATH/src/github.com/NebulousLabs/Sia && make) && npm run sia-kill && npm run sia-import",
```

Comment:
probably best if you undo this change. Seems like a shortcut to you, but would be confusing to newbies.